projects/inkscape/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -eu
+nproc=$(nproc)
+
+### backup flags ###
+
+OLDCFLAGS=${CFLAGS}
+OLDCXXFLAGS=${CXXFLAGS}
+OLDSANITIZER=${SANITIZER}
+OLDCOVERAGE_FLAGS=${COVERAGE_FLAGS}
```

Comment:
COVERAGE_FLAGS are not part of the interface. Do you need these?

Reply:
maybe not, I just wanted to "clean" everything I saw in `env` that could affect the build (cf below)